sample-app/src/main/java/com/squareup/contour/sample/ExpandableBioCard1.kt

```diff
@@ -33,10 +34,15 @@ import com.squareup.contour.ContourLayout
 import com.squareup.picasso.Picasso
 
 @SuppressLint("SetTextI18n")
-class ExpandableBioCard1(context: Context) : ContourLayout(context) {
+class ExpandableBioCard1(context: Context, attributeSet: AttributeSet? = null) : ContourLayout(context) {
```

Comment:
Let's rename the params to `attrs` and pass them to `ContourLayout`'s constructor as well?

Reply:
Same for `ExpandableBioCard2` and `SampleView`.